### DISCUSSION TITLE
Row Spanning implementation with react-table v8

### DISCUSSION THREAD

**Discussion by vladyslavDevInmost** [TanStack/table#5064] [Q&A]
Hi! I'm stuck with the implementation of one feature.
I'd like to build grouped rows, but row-span-look-like. Could you please help me to sort this out with a new v8 api?

<img width="534" alt="Screenshot 2023-09-12 at 19 28 57" src="https://github.com/TanStack/table/assets/140698847/aa2d970f-46bd-4c70-bd83-c49201a29578">

**Comment by ObaidAshiq** [comment]
have u made any progress on this?

**Comment by uginy** [comment]
I have some implementation so far for v.8. If you find it useful, let me know please. 
For 1 st column in a row it works + - ok, but as for me I need to implement row spanning for any kind of columns.
So you can fork this and play. 
https://stackblitz.com/edit/tanstack-table-admbfn
